wandb/cli/cli.py

```diff
@@ -672,6 +672,9 @@ def _clean():
 @click.option("--program", default=None, help="Set sweep program")
 @click.option("--settings", default=None, help="Set sweep settings", hidden=True)
 @click.option("--update", default=None, help="Update pending sweep")
+@click.option(
+    "--convert-flat", is_flag=True, help="Pass in basic config to convert", hidden=True
```

Comment:
nit: "convert-flat" is kind of weird, because it doesn't really convert flat to a nested structure it just injects things under our own keys. I can't think of anything better...